File: Python/Product.py
class Product:
    # Attribute declarations
    __id = ""
    __name = ""
    __brand = ""
    __price = -1
    
    # Constructor
    def __init__(self, id="n/a", name="n/a", brand="n/a", price=-1):
        self.__id = id
        self.__name = name
        self.__brand = brand
        self.__price = price 
    
    # Methods

    # Get methods
    def getID(self):
        return self.__id
    def getName(self):
        return self.__name
    def getBrand(self):
        return self.__brand
    def getPrice(self):
        return self.__price
    
    # Set methods
    def setID(self, id):
        self.__id = id
    def setName(self, name):
        self.__name = name
    def setBrand(self, brand):
        self.__id = id
    def setPrice(self, price):
        self.__id = id

File: PHP/Product.php
<?php
class Product {
    // Private attributes
    private $id = '';
    private $name = '';
    private $brand = '';
    private $price = -1;

    /* Constructor */ 
    public function __construct($id = 'n/a', $name = 'n/a', $brand = 'n/a', $price = -1) {
        $this->id = $id;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
    }

    /* Methods */ 
    // Get methods
    public function getID() { return $this->id; }
    public function getName() { return $this->name; }
    public function getBrand() { return $this->brand; }
    public function getPrice() { return $this->price; }

    // Set methods
    public function setId($id) { $this->id = $id; }
    public function setName($name) { $this->name = $name; }
    public function setBrand($brand) { $this->brand = $brand; }
    public function setPrice($price) { $this->price = $price; }
}

?>
